src/test/java/org/folio/rest/impl/OaiPmhImplTest.java

```diff
@@ -217,6 +218,9 @@ void setUpOnce(Vertx vertx, VertxTestContext testContext) throws Exception {
     logger.info(format("mod-oai-pmh test: Deploying %s with %s", RestVerticle.class.getName(), Json.encode(conf)));
 
     DeploymentOptions opt = new DeploymentOptions().setConfig(conf);
+    idleTimeout = System.getProperty(REPOSITORY_SRS_CLIENT_IDLE_TIMEOUT_SEC);
```

Comment:
After my code suggestions, we are able to get and mock this property via a call to mod-configuration which is more correct and the current test approach uses the mocking of mod-config responses. So, can we address the property from RepositoryConfigUtil.getProperty(...) and mock it via mod-configuration call?

Reply:
It is possible to use it as is as the idle timeout will be taken from system properties if value is not present in request.